src/structures/MessageMentions.js

```diff
@@ -136,12 +133,11 @@ class MessageMentions {
   get members() {
     if (this._members) return this._members;
     if (!this.guild) return null;
-    this._members = new Collection();
-    this.users.forEach(user => {
+    return this.users.reduce((members, user) => {
       const member = this.guild.members.resolve(user);
-      if (member) this._members.set(member.user.id, member);
-    });
-    return this._members;
+      if (member) members.set(member.id, member);
+      return members;
+    }, (this._members = new Collection()));
```

Comment:
I see no benefit in this... if anything making it less readable.. 

CC @iCrawl @kyranet thoughts?

Reply:
No big opinion on this. Can stay like this (changed) for what it's worth.